src/components/common/Button/ButtonStyle.tsx

```diff
@@ -22,12 +24,14 @@ export const ButtonStyle = styled.button<IProps>`
 
   transition: 0.5s;
   cursor: pointer;
-  font-size: 16px;
+  font-size: ${STYLE_CONSTANTS.FONT_SIZE.small};
   font-weight: 700;
 
   :hover {
     background: ${STYLE_CONSTANTS.COLORS.darkYellowGreen};
     color: ${STYLE_CONSTANTS.COLORS.white};
     box-shadow: 5px 5px 5px ${STYLE_CONSTANTS.COLORS.darkGrey};
+    ${({ primary }) =>
```

Comment:
instead of overriding background you can use callback with the ternary operator inside the property
```suggestion
    background:  ${({ primary }) => primary ? STYLE_CONSTANTS.COLORS.slateBlue : STYLE_CONSTANTS.COLORS.darkYellowGreen};
```